src/main/java/com/epam/ta/reportportal/core/integration/bootstrap/DefaultPluginBootstrapper.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.ta.reportportal.core.integration.plugin.bootstrap;
+
+import com.epam.ta.reportportal.core.integration.bootstrap.PluginBootstrapper;
+import com.epam.ta.reportportal.core.integration.plugin.PluginLoader;
+import com.epam.ta.reportportal.dao.IntegrationTypeRepository;
+import com.epam.ta.reportportal.entity.integration.IntegrationType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+
+import static java.util.Optional.ofNullable;
+
+/**
+ * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
+ */
+@Service
+public class DefaultPluginBootstrapper implements PluginBootstrapper {
+
+	private static final Logger LOGGER = LoggerFactory.getLogger(DefaultPluginBootstrapper.class);
+
+	private final IntegrationTypeRepository integrationTypeRepository;
+	private final PluginLoader pluginLoader;
+
+	@Autowired
+	public DefaultPluginBootstrapper(IntegrationTypeRepository integrationTypeRepository, PluginLoader pluginLoader) {
+		this.integrationTypeRepository = integrationTypeRepository;
+		this.pluginLoader = pluginLoader;
+	}
+
+	@Override
+	@PostConstruct
+	public void startUp() {
+		// load and start all enabled plugins of application
+		integrationTypeRepository.findAll()
+				.stream()
+				.filter(IntegrationType::isEnabled)
```

Comment:
Probably we must start disabled plugins as well. In future we can enable it, but we won't have loaded plugin?

Reply:
LoadPluginsJob will load plugins that were enabled later